### PR TITLE
[Backup] fixing workload restore for limited roles

### PR DESCRIPTION
**Pull request by hiaga**
## Description

Fixing SQL/HANA restore for newly added Linked Access Check

## Checklist

- [Yes ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [Yes ] The title of the PR is clear and informative
- [Yes ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [Yes ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [N/A ] If applicable, the changes made in the PR have proper test coverage
- [N/A ] For public API changes to cmdlets:
    - [ N/A] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [N/A ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by hiaga** [comment]
@isra-fel  Can you please review this PR ?

File: src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs [hunk 1]
```diff
@@ -300,7 +300,8 @@ public string GetResourceId()
                 resourceIdentifier.ResourceGroupName = RecoveryPoint.ContainerName.Split(new string[] { ";" }, StringSplitOptions.None)[1];
                 resourceIdentifier.ResourceType = "/VMAppContainer";
```
Comment (isra-fel) [inline src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs:301]:
```suggestion
                resourceIdentifier.ResourceType = "Microsoft.Compute/virtualMachines";
```
and still `return resourceIdentifier.ToString();` should fix your problem

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by hiaga** [comment]
@isra-fel  Great suggestion. I couldn't find the implementation for resourceIdentifier.ToString() - could you point me to it please.
I have made the changes and tested, please review  again.

**Comment by isra-fel** [comment]
> @isra-fel Great suggestion. I couldn't find the implementation for resourceIdentifier.ToString() - could you point me to it please.
> I have made the changes and tested, please review again.

It's in another repo: https://github.com/Azure/azure-powershell-common/blob/yeming/mhsm/src/ResourceManager/Version2016_09_01/Utilities/Models/ResourceIdentifier.cs#L30

**Review by isra-fel — APPROVED** [review]

**Pull request merged**